### DISCUSSION TITLE
Update the Service

### DISCUSSION THREAD

**Discussion by thproflord** [dofire/Floating-Bubble-View#63] [General]
Hi so I was looking a way of implementing a way for updating the bubble after created, If i send the startForegrounservice after it is already created, of course it does not call the on Create of the service again, how can i send a signal or something to the service tu run a function from any activity to update its view?

**Comment by dofire** [comment]
@thproflord You can send the intent data with `startForegroundService` and handle logic inside `onStartCommand()`
